src/modules/settings/BlockedSkills.svelte

```diff
@@ -0,0 +1,226 @@
+<script>
+import getValueJSON from '../system/getValueJSON';
+import setValue from '../system/setValue';
+import {
+  checkSkill,
+  clearCheckedSkills,
+  getCheckedSkills,
+  submitSkillChanges,
+} from './blockedSkills';
+
+let blockedSkillLists = getValueJSON('blockedSkillLists');
+let newListName = '';
+let error = false;
+let listName = '';
+
+function checkForDuplicates(list) {
+  return blockedSkillLists.some((l) => {
+    if (l.name === list.name) {
+      return false;
+    }
+    if (l.skills.length !== list.skills.length) {
+      return false;
+    }
+    return l.skills.every((s, i) => s === list.skills[i]);
+  });
+}
+
+const loadListChecks = [
+  [
+    (list) => !list,
+    (list) => `Unable to find list "${list.name}". Try refreshing the page.`,
+  ],
+];
+
+const updateListChecks = [
+  [
+    (list) => !blockedSkillLists.some((l) => l.name === list.name),
+    (list) => `Cannot find list named "${list.name}". Try refreshing the page.`,
+  ],
+  [
+    (list) => checkForDuplicates(list),
+    () => 'You already have a list with these skills.',
+  ],
+];
+
+const newListChecks = [
```

Comment:
create a function that calls checkForErrors with this array
blockedSkillLists would be a param for the new function
passed as arg for checkForErrors
moved to import